### PR TITLE
render examples with 'njk' language as nunjucks templates and display result

### PR DESCRIPTION
**Pull request by carljm**

### CODE REVIEW

**Comment by jgerigmeyer** [comment]
I'm fine with merging this. @davisagli or @carljm?

**Comment by carljm** [comment]
:shipit:

**Comment by davisagli** [comment]
This needs a mention in documentation. Where's the right place to add that? And is there a changelog/release notes?

**Comment by mirisuzanne** [comment]
@davisagli there isn't yet, but it would be a good idea. The README might be a good place to start for basic feature documentation, until it gets too big?

**Comment by jgerigmeyer** [comment]
If only there was a better way to auto-generate some documentation...

**Comment by davisagli** [comment]
@mirisuzanne please review docs (docs for the other bits are left as an exercise...) and feel free to merge.

File: README.md [hunk 1]
```diff
@@ -22,3 +18,70 @@ user-experience and code patterns:
...
+
+```
+npm install sassdoc-theme-herman
+```
+
+See the [SassDoc documentation](http://sassdoc.com/getting-started/)
+for how to install SassDoc and run it via various build tools.
+
+Herman is a theme for SassDoc,
+so you'll want to specify `sassdoc-theme-herman`
```
Comment (mirisuzanne) [inline README.md]: I think they only need to specify `herman` - @jgerigmeyer is that right?

Comment (jgerigmeyer) [inline README.md]: correct

**Pull request merged**